### DISCUSSION TITLE
How to you set amount in paymentintent?

### DISCUSSION THREAD

**Discussion by GMerakis** [flutter-stripe/flutter_stripe#550] [General]
I looked at examples https://github.com/flutter-stripe/flutter_stripe/blob/08b7a1e9896fc4aa12eb1d4734cf2c72fe9cd8a7/example/lib/screens/card_payments/webhook_payment_screen.dart#L108

All it has is creating payment intent with currency and items.

In stripe_payment we can just create payment intent with current & amount which are required fields for stripe.

**Comment by remonh87** [comment]
you need to set the amount on the API  see https://stripe.com/docs/api/payment_intents . Never create the payment intent  from the client but ask your backend server to create it.

**Reply by GMerakis** [reply]
ah I see. Thanks!
